src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400px;
+}
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900px;
+}
```

Comment:
You imported 2 fonts with the same name and when you try to use the font in styles browser always will be take that font which is written below.
So for Heavy font change the name in import and in focus.

And don't forget to check for any font weight that affects the font while the bar is out of focus

Reply:
Finally all tests are passed.